src/persist/src/indexed/encoding.rs

```diff
@@ -1166,28 +1139,30 @@ mod tests {
         );
 
         // Not sorted by key
-        let b = BlobTraceBatchPart {
-            desc: u64_desc(0, 2),
-            index: 0,
-            updates: columnar_records(vec![update_with_key(0, "1"), update_with_key(1, "0")]),
-        };
-        assert_eq!(
-            b.validate(),
-            Err(Error::from(
-                "unsorted: (\"1\", \"\", 0) was before (\"0\", \"\", 1)"
-            ))
-        );
+        // TODO: Re-activate once we add part sorting in the new API.
```

Comment:
lets delete this instead of comment it out. we can always pull it out of git history and in the meantime it won't be cluttering things up